docs/source/conf.py

```diff
@@ -38,43 +40,40 @@
     'torchvision',
     'torchvision.models',
     'torchtext',
-    'torch_stft',
-    'pytorch_lightning',
-    'h5py',
-    'kaldi_io',
-    'transformers',
-    'transformers.tokenization_bert',
-    'apex',
-    'ruamel',
-    'frozendict',
-    'inflect',
-    'unidecode',
-    'librosa',
-    'soundfile',
-    'sentencepiece',
-    'youtokentome',
-    'megatron-lm',
-    'numpy',
-    'dateutil',
-    'wget',
-    'scipy',
-    'pandas',
-    'matplotlib',
```

Comment:
ok, this makes me wonder... don't we need to mock up those modules anymore?

Reply:
This mockup is now done below by parsing all of the requirements*.txt from nemo and setting them all up for mock imports

Reply:
oooh, right, I stopped just before this.... this is awesome!!